plugins/drivers/testutils/dns_testing.go

```diff
@@ -16,7 +16,11 @@ func TestTaskDNSConfig(t *testing.T, driver *DriverHarness, taskID string, dns *
 		caps, err := driver.Capabilities()
 		require.NoError(t, err)
 
-		isolated := (caps.FSIsolation != drivers.FSIsolationNone)
+		// FS isolation is used here as a proxy for network isolation.
```

Comment:
Our driver capabilities advertise network isolation modes. Is that not feasible here?https://www.nomadproject.io/docs/internals/plugins/task-drivers#capabilities-capabilities-error

Reply:
I considered but they advertise the network capabilities for isolation, not necessarily their default ones.

Reply:
Ah, makes sense 👍

Reply:
Also, I think these network capabilities are a bit confusing, as they indicate Nomad's network management, rather than driver behavior necessarily.  For example, Setting `NetIsolationModeHost` for Docker means that Nomad will not create a network namespace, but Docker will still by default create a network namespace.